packages/api-explorer/package.json

```diff
@@ -72,6 +71,10 @@
     "react-router-dom": "^5.1.2",
     "redux": "^4.0.5",
     "styled-components": "^5.2.1",
-    "ts-jest": "^26.2.0"
+    "ts-jest": "^26.2.0",
+    "webpack-merge": "^5.7.3",
+    "webpack-bundle-analyzer": "^4.2.0",
+    "webpack-cli": "^4.6.0",
+    "webpack-dev-server": "^3.11.2"
```

Comment:
can these be `devDependencies`?

Reply:
Yes. Dammit I thought I had moved them

Reply:
I did move them, but in this package devDependencies come before dependencies. In others they are reversed! Argh!

Reply:
Caught by inconsistency